V5_MIGRATION.md

```diff
@@ -171,3 +171,9 @@ venmoDriver.tokenizeVenmoAccount(with: venmoRequest) { (venmoAccountNonce, error
   // transact with nonce on server
 }
 ```
+
+## BraintreeDataCollector
```

Comment:
```suggestion
## Data Collector
```

The other sections are named more generically (e.g., `Card`, `Venmo`) so maybe we should do the same here.